recipes/py3nvml/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "py3nvml" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/fbcotter/py3nvml/archive/{{ version }}.tar.gz"
+  sha256: "6f111b8bc460228ca5c5b430f80d0dbc27b11e8ad5a589bd0be51fae879bb75e"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
```

Comment:
I think this recipe can be `noarch`, please add `noarch: python`.

Reply:
In this case it can't due to: `- enum34      # [py < 34]`